src/SoftRobots/component/constraint/model/CableModel.h

```diff
@@ -158,15 +185,22 @@ class SOFA_SOFTROBOTS_API CableModel : virtual public SoftRobotsConstraint<DataT
 
     void checkIndicesRegardingState();
     void initActuatedPoints();
+    void initCableActionAreas();
+    void computePointsActionArea();
+    unsigned int computeClosestIndice(const Coord& position);
+    void getPositionFromTopology(Coord& position, const int& index);
```

Comment:
Not a big deal, but it is usually faster to pass primitive types by value rather than by const-ref. Also, `getPX` accepts a `sofa::Index`, so it's preferable to continue using a `sofa::Index` rather than an `int`.
```suggestion
    void getPositionFromTopology(Coord& position, sofa::Index index);
```